nats/src/test/kotlin/rys/ajaxpetproject/nats/NatsControllersUT.kt

```diff
@@ -31,23 +40,26 @@ import rys.ajaxpetproject.request.update.create.proto.ChatUpdateRequest
 import rys.ajaxpetproject.request.update.create.proto.ChatUpdateResponse
 import rys.ajaxpetproject.service.ChatService
 import rys.ajaxpetproject.service.impl.ChatServiceImpl
+import rys.ajaxpetproject.service.impl.MessageServiceImpl
+import rys.ajaxpetproject.service.impl.UserServiceImpl
 import rys.ajaxpetproject.subjects.ChatSubjectsV1
 import java.time.Duration
 
 @SpringBootTest(classes = [NatsTestConfiguration::class])
 @ContextConfiguration(
     classes = [
-        ChatRepository::class, ChatServiceImpl::class,
+        ChatRepositoryImpl::class, ChatServiceImpl::class,
         NatsChatCreationController::class,
         NatsChatDeleteController::class,
-        NatsChatFindAllController::class,
-        NatsChatFindOneController::class,
-        NatsChatUpdateController::class,
-        NatsControllerConfigurerPostProcessor::class
-        ]
+        NatsChatFindAllController::class, MessageRepository::class, UserRepository::class,
+        NatsChatFindOneController::class, MessageServiceImpl::class, UserServiceImpl::class,
+        NatsChatUpdateController::class, SecurityConfiguration::class,
+        NatsControllerConfigurerPostProcessor::class,
+
+    ]
 )
 @ActiveProfiles("testing")
-class NatsControllersTests {
+class NatsControllersUT {
```

Comment:
suffix `UT` means `unit test`? If yes then it's not true. It's intergration test

Reply:
Sorry for the confusion. 
Fixed earlier in another PR